app/pages/resource/resource-header/ResourceHeader.js

```diff
@@ -47,6 +51,21 @@ function ResourceHeader({
   const typeName = resource.type ? resource.type.name : '\u00A0';
   const distance = formatDistance(resource.distance);
 
+  let linkButton;
+  if (outlookLinkExists) {
+    linkButton = (<Button
```

Comment:
Linter errors prevent build

Reply:
Ups.. Will fix asap.

Reply:
Had some unforeseen delays during the day, but fixed all lint problems cause by my code.